templates/epic.xml.jinja2

```diff
@@ -44,15 +44,15 @@
       The ip6 (or other ip) defines should be included first.
       These files have only a define tags.
   </documentation>
-    <include ref="ip6/definitions.xml" />
-    <include ref="ip6/far_forward/fields_{{ pbeam | default("275", true) }}.xml" />
-    <include ref="compact/definitions.xml" />
+    <include ref="${BEAMLINE_PATH}/ip6/definitions.xml" />
+    <include ref="${BEAMLINE_PATH}/ip6/far_forward/fields_{{ pbeam | default("275", true) }}.xml" />
+    <include ref="${DETECTOR_PATH}/compact/definitions.xml" />
   </define>
 
   <includes>
-    <gdmlFile ref="compact/elements.xml"/>
-    <gdmlFile ref="compact/materials.xml"/>
-    <file     ref="compact/optical_materials.xml"/>
+    <gdmlFile ref="${DETECTOR_PATH}/compact/elements.xml"/>
+    <gdmlFile ref="${DETECTOR_PATH}/compact/materials.xml"/>
+    <file     ref="${DETECTOR_PATH}/compact/optical_materials.xml"/>
```

Comment:
I have the same reservation about using DETECTOR_PATH as in https://github.com/eic/ip6/pull/9#issuecomment-1200526270. Would be interested to hear your line of thinking when you made this.

Reply:
I just had to add those `${DETECTOR_PATH}` when creating my own composition for the EPIC detector by first copying an existing xml from a package to my local directory. Having this PR as is would be useful for that case.